rpcserver.go

```diff
@@ -1328,14 +1328,19 @@ func (r *rpcServer) ChannelBalance(ctx context.Context,
 		return nil, err
 	}
 
-	var balance btcutil.Amount
+	var pendingBalance, balance btcutil.Amount
 	for _, channel := range channels {
-		if !channel.IsPending {
+		if channel.IsPending {
+			pendingBalance += channel.LocalCommitment.LocalBalance.ToSatoshis()
```

Comment:
FWIW, this block will only be executed if the channel is pending-open, computing the pending-close balance is a lot more involved. Just to avoid confusion, let's make this `pendingOpenBalance`.

Reply:
changed, thanks!